packages/drivers/routerlicious-urlResolver/.eslintrc.cjs

```diff
@@ -23,18 +23,5 @@ module.exports = {
 				ignore: [/.*routerlicious-urlResolver\.spec\.ts/],
 			},
 		],
-
-		// This library is used in the browser, so we don't want dependencies on most node libraries.
-		"import/no-nodejs-modules": ["error"],
 	},
-	overrides: [
-		{
-			// Rules only for test files
-			files: ["*.spec.ts", "src/test/**"],
-			rules: {
-				// Test files are run in node only so additional node libraries can be used.
-				"import/no-nodejs-modules": ["error", { allow: ["assert"] }],
```

Comment:
This seems slightly stricter than the one in minimal-deprecated (only allowing assert) - would we want to strengthen the shared config to only allow "assert" too?

Reply:
For test files, I think general node usage is okay.

Reply:
The default is a lot more forgiving. It says any Node use is okay for tests which is intentional and fine. I didn't see value in having this stricter override.

Reply:
This assumes we run all our tests in node, which is not true or it shouldn't be, at least.

Regardless, any change should be in the shared config anyway.